susemanager/src/mgr_bootstrap_data.py

```diff
@@ -352,6 +352,86 @@
     "libunwind",
 ]
 
+PKGLISTUBUNTU1604 = [
+    "libsodium18",
```

Comment:
Please notice that some of the packages listed here have a different name in the specfile in OBS, e.g. `libsodium`
Would that be a problem?

Reply:
It's not a problem. In the repo the binaries are called `libsodium18` and `libsodium23`. In the `control` file the name is set explicitly to `Package: libsodium23` for e.g.